rpm/centos-7/docker-ce.spec

```diff
@@ -1,217 +0,0 @@
-Name: docker-ce
-Version: %{_version}
-Release: %{_release}%{?dist}
-Epoch: 3
-Summary: The open-source application container engine
-Group: Tools/Docker
-License: ASL 2.0
-Source0: engine.tgz
-Source1: cli.tgz
-URL: https://www.docker.com
-Vendor: Docker
-Packager: Docker <support@docker.com>
-
-# DWZ problem with multiple golang binary, see bug
-# https://bugzilla.redhat.com/show_bug.cgi?id=995136#c12
-%global _dwz_low_mem_die_limit 0
-%global is_systemd 1
-%global with_selinux 1
-
-BuildRequires: make
-BuildRequires: cmake
-BuildRequires: gcc
-BuildRequires: git
-BuildRequires: glibc-static
-BuildRequires: libtool-ltdl-devel
-BuildRequires: libseccomp-devel
-BuildRequires: device-mapper-devel
-BuildRequires: btrfs-progs-devel
-BuildRequires: pkgconfig(systemd)
-BuildRequires: pkgconfig(libsystemd-journal)
-
-# required packages on install
-Requires: /bin/sh
-Requires: container-selinux >= 2.9
-Requires: libseccomp >= 2.3
-Requires: iptables
-Requires: libcgroup
-Requires: systemd-units
-Requires: tar
-Requires: xz
-
-# Resolves: rhbz#1165615
-Requires: device-mapper-libs >= 1.02.90-1
-
-# conflicting packages
-Conflicts: docker
-Conflicts: docker-io
-Conflicts: docker-engine-cs
-Conflicts: docker-ee
-
-# Obsolete packages
-Obsoletes: docker-ce-selinux
-Obsoletes: docker-engine-selinux
-Obsoletes: docker-engine
-
-%description
-Docker is is a product for you to build, ship and run any application as a
-lightweight container.
-
-Docker containers are both hardware-agnostic and platform-agnostic. This means
-they can run anywhere, from your laptop to the largest cloud compute instance and
-everything in between - and they don't require you to use a particular
-language, framework or packaging system. That makes them great building blocks
-for deploying and scaling web apps, databases, and backend services without
-depending on a particular stack or provider.
-
-%prep
-%setup -q -c -n src -a 1
```

Comment:
the `-a 1` was also not in the current version of the spec (but didn't have time yet to search what it does / why it was there 😅)